### PR TITLE
[FOR REVIEW] Rewrite open bounties to better match current templates and guidelines. 

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: bounties/BA062302.md [hunk 1]
```diff
@@ -22,29 +22,35 @@ contacts:
...
-The impact of individuals’ contributions will consider effort, time saved, and severity (eg, “without this contribution, the vote would’ve been invalid”).
+Catching one of the following issues may lead to compensation under this bounty, depending on the time saved, the importance of the RPIP and the severity of the issue.
+* **High Severity**: This issue would invalidate a vote or necessitate immediate rework.
+* **Medium Severity**: This issue requires clarification during an ongoing vote or necessitates rework in the near future. 
+* **Low Severity**: This issue can be rectified by RPIP editors after the vote, encompassing general aspects like spelling, grammar, formatting, and similar concerns.
 
-**Expiry**
-* This bounty expires after 3 months.
+**Expiry**  
+* This bounty expires on 2023-02-28.
```
Comment (LongForWisdom) [inline bounties/BA062302.md]: Update to 2024.

File: bounties/BA062304.md [hunk 2]
```diff
@@ -4,58 +4,66 @@ title: Rocket Pool Scorecard
 proposal-author: Valdorff
 creation-date: 2023-11-28
 external-status: open
-summary: Construct or Review a Rocket Pool scorecard that can be added to the Rocket Pool website (or similar) in the same theme as the [Lido Scorecard](https://lido.fi/scorecard).
+summary: Write or Review the content for a Rocket Pool scorecard in the same theme as the [Lido Scorecard](https://lido.fi/scorecard).
 reward:
   unit: USD
-  max: 15000
+  max: 8000
   min: 2000
```
Comment (LongForWisdom) [inline bounties/BA062304.md]: Update min amount to match text.

File: bounties/BA082303.md [hunk 3]
```diff
@@ -21,23 +21,28 @@ contacts:
...
-## Optionals
-* No optionals.
+## Required Milestones
 
-## Further Notes
+### Milestone A - Treegen Implementation
+**Payout:** Up to 1,500 USD
 
-The reward is set at $1500 per new treegen specification. The reward for each specification will be split between bounty hunters that meet the listed requires. Open source treegen implementations will be given twice the weight when splitting the reward. 
+* You must create or update an independent version of treegen with a the rewards specification without duplicating the new code from the Rocket Pool development team.
```
Comment (LongForWisdom) [inline bounties/BA082303.md]: Fix typo

File: bounties/BA082402.md [hunk 4]
```diff
@@ -23,30 +23,50 @@ contacts:
...
+* The product must have at least 100 minipools as defined in the Rocket Pool contracts.
+* Must be completed by the same party to complete Milestone A, unless that party has explicitly granted consent for completion by another party.
+
+### Milestone C - Scaling
+**Payout:** $60,000  
+
+* You must provide evidence of a functioning and scalable product.
+* The product must be meet the the specification requirements laid out in Milestone A.
+* Block proposals made by users of the Rocket Pool contracts within the product must exceed 5% of the proposals made by users of Allnode's Rocket Pool service over the period of a week. 
+* The product must be meet the the specification requirements laid out in Milestone A.
```
Comment (LongForWisdom) [inline bounties/BA082402.md]: Remove duplicated line

File: bounties/BA082402.md [hunk 5]
```diff
@@ -23,30 +23,50 @@ contacts:
...
-* **Milestone 2**: 5% the size of allnodes (within RP) – measured by block proposals across a week
+The service need not be identifical to AllNodes. For example, it could require running something very lightweight on a home computer, which could be run on a primary machine (as opposed to dedicated hardware).
 
-Bounties for milestones are awarded according to the following scheme:
-* **Milestone 0**: First to write a detailed project specification receives $5,000
-* **Milestone 1**: $20,000
-* **Milestone 2**: $60,000
+## Verification
+* The GMC must approve each milestone.
+* Please contact the GMC Administrator to arrange reivew of each milestone as it is completed.
```
Comment (LongForWisdom) [inline bounties/BA082402.md]: Fix typo

**Comment by shfryn** [comment]
Reviewed on call, and reviewed after - some will be updated shortly as discussions evolve.

File: bounties/BA062304.md [hunk 6]
```diff
@@ -4,58 +4,66 @@ title: Rocket Pool Scorecard
...
+### Milestone C - Review
+**Payout:** 100 - 2,000 USD
 
-Bounties for milestones are awarded according to the following scheme:
-* **Milestone 1**: $5000
-* **Milestone 2**: $2,000 - $8,000 is allocated to be split among reviewers and content creators. The amount of compensation is based on the quality of the content.
+* Reviews may cover both copy editing and proof-reading. 
+* Reviewers should ensure the factual accuracy of the scorecard.
+* Reviewers should comment on the content selected for inclusion.
+* Reviewers may suggest content for inclusion that they feel is missing.
```
Comment (Valdorff) [inline bounties/BA062304.md:53]: I'd probably add a bit more flexibility here:
`Reviewers may offer any suggestions around the content, including: addition of content that they feel is missing, removal of content they feel is low-value, changes to content presentation, etc.

Comment (LongForWisdom) [reply bounties/BA062304.md:53]: Fair.

**Pull request merged**